os/booting-on-azure.md

```diff
@@ -136,11 +136,11 @@ Now that you have a machine booted it is time to play around. Check out the [Cor
 [azurecli-heading]: #via-the-cross-platform-cli
 [cloud-config-docs]: {{site.baseurl}}/os/docs/latest/cloud-config.html
 [cloud-config-heading]: #cloud-config
-[docs]: {{site.baseurl}}/docs
```

Comment:
They do, but these sources should stand on their own. The sync script can rewrite `coreos.com` to `{{site.baseurl}}` to make sure staging continues to work.